subprojects/distributions/distributions.gradle

```diff
@@ -90,6 +84,7 @@ task allZip(type: Zip) {
 }
 
 task binZip(type: Zip) {
+    outputs.cacheIf { true }
     classifier = 'bin'
     into(zipRootFolder) {
         with binDistImage
```

Comment:
Yes, that is something we already fixed. Especially, any changes to the files comprising the `binZip` would cause the task to be out-of-date.